msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -955,14 +955,15 @@ The `extensions` property specifies Outlook Add-ins within an app manifest and s
 |`autoRunEvents`| Array | | | Defines the event-based activation extension point. |
 |`alternates`| Array | | | Specifies the relationship to alternate existing Microsoft 365 solutions. It's used to hide or prioritize add-ins from the same publisher with overlapping functionality. |
 |`audienceClaimUrl`| String | 2048 characters | | Specifies the URL for your extension and is used to validate Exchange user identity tokens. For more information, see [inside the Exchange identity token](/office/dev/add-ins/outlook/inside-the-identity-token)|
+|`contextMenus`| Array | | | Specifies the context menus for your extension. A context menu is a shortcut menu that appears when you right-click in the Office UI. |
```

Comment:
Is it 'right-click' within the Office UI?

Reply:
Not sure about using "within" but updated instruction according to [MS writing guidance](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/right-click)

Reply:
Opting to keep "in" per our editor and [guidance](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/mouse-mouse-interaction-terms)